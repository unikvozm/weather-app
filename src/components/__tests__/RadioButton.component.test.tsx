import React from 'react';
import renderer from 'react-test-renderer';
import {RadioButtonComponent} from '../RadioButton.component';

test('RadioButtonComponent renders correctly', () => {
  const tree = renderer
    .create(
      <RadioButtonComponent
        data={[{value: 'value', text: 'text'}]}
        onValueChange={jest.fn}
        initialValue="value"
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
