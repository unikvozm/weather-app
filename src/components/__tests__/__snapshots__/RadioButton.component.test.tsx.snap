// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButtonComponent renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 14,
          "marginRight": 15,
        }
      }
    >
      text
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "borderColor": "#3EEFD8",
          "borderRadius": 50,
          "borderWidth": 2,
          "height": 20,
          "justifyContent": "center",
          "opacity": 1,
          "width": 20,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#3EEFD8",
            "borderRadius": 50,
            "height": 10,
            "width": 10,
          }
        }
      />
    </View>
  </View>
</View>
`;
